### PR TITLE
integration test for lib/util/union_find.py

### LINKED ISSUE #143: Create integration tests for modules in lib/util
Order of completion:    
 - [x] json_encoder.py 
 - [x] graph.py
 - [x] preprocess.py
 - [x] union_find.py
 - [ ] layout.py

### PR DESCRIPTION
**Pull request by tejalwakchoure**
Fixes #143 
## Notify reviewers
@prasadtalasila

### CODE REVIEW

File: test/integration_test/lib/util/test_union_find.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from lib.util.union_find import *
+import pytest
+
+def test_union_find(self):
```
Comment (prasadtalasila) [inline test/integration_test/lib/util/test_union_find.py]: The class name should be `TestUnionFind` and have one test method for each method that is getting tested.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]

**Comment by prasadtalasila** [comment]
@tejalwakchoure  Please add tests to make sure you have all the code lines covered by tests (near 100% code coverage).

**Comment by tejalwakchoure** [comment]
@prasadtalasila Updated. The lines that are still not covered pertain to private functions so I am not sure how to test them.

**Comment by prasadtalasila** [comment]
@tejalwakchoure The lines that are not covered by the test are: [39 to 41](https://codecov.io/gh/DeveloperCAP/MLCAT/pull/161/changes#L39). These lines belong to make_set() function; this function is not private. You can invoke make_set() such that the conditions on lines 34 and 36 get bypassed and you get a chance to execute lines 39 to 41.

**Comment by prasadtalasila** [comment]
Looks good.

**Review by prasadtalasila — APPROVED** [review]

**Pull request merged**